tket/tests/Placement/test_Placement.cpp

```diff
@@ -19,723 +19,135 @@
 #include "Placement/Placement.hpp"
 
 namespace tket {
-namespace test_Placement {
```

Comment:
Have all the tests deleted from here moved to other files?

Reply:
They have not all been moved over identically, but I've made sure that all the tests covering basic functionality are matched (in a neater fashion), and that any tests that were clearly there to catch some known bug at some point (i.e. a peculiar architecture or circuit construction) were moved over.